### PR TITLE
Fixing storybook build error

### PR DESCRIPTION
**Pull request by ryanml**
Fixes brave/brave-browser#7067

This needed to be updated after latest grant refactor

## Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [x] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [x] Tagged reviewers and labelled the pull request as needed.
- [ ] Request a security/privacy review as needed.
- [x] Add appropriate QA labels (QA/Yes or QA/No) to include the closed issue in milestone
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protection-Mode
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Test Plan:
1. Run `npm run storybook`
2. Confirm storybook builds with no errors

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

**Comment by NejcZdovc** [comment]
@ryanml should we add this to CI? As it's really easy to forget about updating storybook especially if you are not updating any UI

**Comment by ryanml** [comment]
@NejcZdovc - I agree, the one issue with adding it to CI is that the command does take awhile, and would slow down the Travis job considerably. Seeing as most commits to brave-core would not affect the storybook. We could make it run dynamically based on whether the correct folder under components is updated, though Travis doesn’t seem to offer a way to do this. So we can: 

A: Make this run as a part of Jenkins, which we can check the diff and decided whether to run it

B: More simply, add another checkbox to the PR template which the pull requester would confirm whether they verified storybook if needed

**Comment by petemill** [comment]
Considering the parallel Jenkins portion of the build takes a few hours, I don't see an issue if the Travis portion takes a few more minutes. The aim anyway is to get a storybook deploy Url for each PR, which would require a build.
Perhaps GitHub actions can be utilized to conditionally run the build depending on the changes paths, but I'm not sure it's necessary.

File: components/brave_rewards/resources/ui/stories/wallet.tsx [hunk 1]
```diff
@@ -72,19 +72,19 @@ storiesOf('Rewards/Wallet/Desktop', module)
         ]}
         grants={showGrant ? [
           {
-            tokens: '8.0',
-            expireDate: '7/15/2018',
-            type: 'ugp'
+            amount: 2.5,
+            expiresAt: '1574451334789',
+            type: 1
```
Comment (petemill) [inline components/brave_rewards/resources/ui/stories/wallet.tsx]: I think this is an aside, but could / do we have a enum-like thing for this type value. For example GrantType.SOMETHING? (Which would evaluate to 1.

**Comment by ryanml** [comment]
Note: All Pipeline steps green, failure unrelated. This only touches storybook code

**Pull request merged**